### ISSUE TITLE
How to set the category_id  when submitting result to server.

### ISSUE THREAD

**Issue by YuHengsss** [epic-kitchens/annotations#30]
Hi, 
Thanks for this great dataset.

I find your pretrained model provided in https://data.bris.ac.uk/data/dataset/gw5q146a9dqd2t82jhu8noe3m with 290 classes. But I can not figure out how to convert the id in class map of this pretrained model to category_id. 

E.g. bag with id 19 in class map of pretrained model but it has id 20 in EPIC_noun_classes.csv. When submitting to the test server, what should be the category_id of the bag?

**Comment by dimadamen** [comment]
The model you are referring to here is for EPIC-KITCHENS-100 (the second and larger version of the dataset). The set of classes have been updated with new/diff IDs. You can find the classes for this model here:
https://github.com/epic-kitchens/epic-kitchens-100-annotations/blob/master/EPIC_100_noun_classes.csv

We do not have a model pre-trained for EPIC-KITCHENS-55.

Note that you cannot use this model for EPIC-KITCHENS-55 as it is trained on the test set in this case.

**Comment by YuHengsss** [comment]
Thanks for your kindly reply. But I am still confused about how to decide the category_id for EPIC-KITCHENS-55 Detection challenge. There are 295 classes for training set as mentioned in https://github.com/epic-kitchens/epic-kitchens-55-annotations/issues/9. Is their category_id same as the list provided by willprice in the issue 9? 
   
Looking forward to your reply.

**Comment by dimadamen** [comment]
For the detection challenge, the category_id is from EPIC-KITCHENS-55 EPIC_noun_classes.csv
All the classes are evaluated, even if they are not in the training set (i.e. zero-shot).
You do not need to worry about the classes for the training set specifically. The predictions for the category ID should match: https://github.com/epic-kitchens/epic-kitchens-55-annotations/blob/master/EPIC_noun_classes.csv
